.travis.yml

```diff
@@ -34,9 +34,11 @@ install:
   - glide "-home" $GLIDE_HOME install
 
 script:
+  - bin/check_vet
+  - bin/builder_run bin/check_lint
```

Comment:
Why do we need to run lint in builder?

Reply:
I found that Glide dont support for DEV dependencies for developer environment.
So decision for DEV dependencies now is:
- all tools documented&frozen in `bin/builder_docker/Dockerfile`
- developer can still install them in local machine